packages/fx-core/src/plugins/resource/aad/utils/telemetry.ts

```diff
@@ -15,11 +15,11 @@ export class TelemetryUtils {
     eventName: string,
     properties?: { [key: string]: string },
     measurements?: { [key: string]: number }
-  ) {
+  ): void {
```

Comment:
It is suggested that add a success property value "yes" for successful result and "no" for failed result.

Reply:
Fixed. Thanks!